crates/luminol/Cargo.toml

```diff
@@ -46,12 +46,15 @@ luminol-term.workspace = true
 # luminol-windows = { version = "0.1.0", path = "../luminol-windows/" }
 # luminol-tabs = { version = "0.1.0", path = "../luminol-tabs/" }
 
+
 poll-promise.workspace = true
 
 camino.workspace = true
 
 strum.workspace = true
 
+tokio = "1.33" # *sigh*
```

Comment:
You don't need to rope in a Tokio runtime just to use reqwest. I only added reqwest because I couldn't get surf to compile in web builds without downgrading wasm-bindgen (surf depends on stdweb which is affected by https://github.com/koute/stdweb/issues/427, and eframe requires wasm-bindgen >= 0.2.87) and I don't think the web version of reqwest needs Tokio, so you can switch back to surf in native builds only if you want.

Reply:
I'd actually like to stick to reqwest because it performs significantly faster than surf it seems